### PR TITLE
build(docs-infra): upgrade `@angular/*` to latest versions

### PR DESCRIPTION
**Pull request by gkalpak**
~~Sits on top of #26391 in order to pass CI.~~ _#26391 has been merged_

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -73,17 +73,17 @@
...
-    "@angular/common": "^7.0.0-rc.0",
-    "@angular/core": "^7.0.0-rc.0",
-    "@angular/elements": "^7.0.0-rc.0",
-    "@angular/forms": "^7.0.0-rc.0",
-    "@angular/material": "6.0.2",
-    "@angular/platform-browser": "^7.0.0-rc.0",
-    "@angular/platform-browser-dynamic": "^7.0.0-rc.0",
-    "@angular/router": "^7.0.0-rc.0",
-    "@angular/service-worker": "^7.0.0-rc.0",
+    "@angular/animations": "^7.0.0-rc.1",
```
Comment (petebacondarwin) [inline aio/package.json]: So someone bumped the animations version without updating the rest? 😱

File: aio/package.json [hunk 2]
```diff
@@ -163,4 +163,4 @@
     "xregexp": "^4.0.0",
     "yargs": "^7.0.2"
   }
-}
+}
```
Comment (petebacondarwin) [inline aio/package.json]: 🤔

File: aio/scripts/_payload-limits.json [hunk 3]
```diff
@@ -3,9 +3,9 @@
     "master": {
       "uncompressed": {
         "runtime": 3173,
-        "main": 483844,
-        "polyfills": 53920,
-        "prettify": 14913
+        "main": 494475,
```
Comment (petebacondarwin) [inline aio/scripts/_payload-limits.json]: Is 10Kb increase acceptable here?

**Review by petebacondarwin — APPROVED** [review]

File: aio/scripts/_payload-limits.json (hunk 3 above)
Comment (gkalpak) [reply aio/scripts/_payload-limits.json]: ❔

Comment (gkalpak) [reply aio/scripts/_payload-limits.json]: Yes, I know 😅. Sorry for my confusing use of emoji.
I was (unsuccessfully) trying to express the fact that I too wonder if it is an acceptable increase. (Mainly for @IgorMinar to decide.)

Comment (IgorMinar) [reply aio/scripts/_payload-limits.json]: those 10kb are coming from CDK according to https://github.com/angular/angular/pull/26059/files#r223122757 - specifically from the a11y improvements. I don't think we'll have a away to treeshake that stuff away :-(

**Review by IgorMinar — APPROVED** [review]

**Comment by gkalpak** [comment]
@IgorMinar, should I mark this for merge or have you deliberately not marked it?

**Pull request closed without merging**